lounge/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+lounge:
+  image: thelounge/lounge:latest
```

Comment:
Pulling from npm is a bit better than from GitHub indeed, but my worries were more about pulling a tag versus pulling `master`. As long as we **do not** pull latest `master`, it's fine.